src/screens/Drawer/CustomDrawer.tsx

```diff
@@ -0,0 +1,183 @@
+                </View>
+
+                <TouchableOpacity onPress={() => props.navigation.closeDrawer()} style={{ marginTop: 20 }}>
+                    <AntDesign name="close" size={24} color="black" />
+                </TouchableOpacity>
+            </LinearGradient>
+        </View>
+        
+
+        <View style={stylesBalance.balance}>
+            {!loading ? (
+          
+                <View> 
+                    <Text style={stylesBalance.text}>Current Balance</Text>
+                    <Text style={stylesBalance.ksh}>Ksh {kshBalance}</Text>
+                    <Text style={stylesBalance.cUSD}>{cUSDBalance} cUSD</Text>
+                </View> 
+                
+            
+            ) : (
+            <View style={stylesBalance.loadingDiv}>
+                <Progress.CircleSnail color={['black']} size={23} thickness={2}/>
+                <Text style={stylesBalance.loadingMessage}>{loadingMessage}</Text>
+            </View>
+            )}
+        </View>
+
+        <DrawerContentScrollView {...props}>
+           <DrawerItemList {...props} />
+          <Text style={styles.versionNumber}>Version 2.0.1</Text>
+        </DrawerContentScrollView>
+        
+      </SafeAreaView>
+    );
+  }
+
+  const styles = StyleSheet.create({
+    container: {
+      width: SIZES.width * 0.8,
+      height: SIZES.height,
+      backgroundColor: COLORS.menuBackground,
+      alignItems: "flex-start",
+      justifyContent: "center",
+    },
+    header: {
+        height: "auto",
+        width: SIZES.width * 0.8,
+        flexDirection: "row",
+        justifyContent: "space-between",
+        paddingLeft: 15,
+        paddingTop: 25,
+    },
+    identiconImg: {
+        width: 48,
+        height: 48,
+        borderRadius: 24,
+        backgroundColor: COLORS.white,
+      },
+    phoneCountryFlag: {
+        width: 16,
+        height: 11,
+        marginTop: 20,
+        backgroundColor: COLORS.white,
+      },
+    phoneNumberTest: {
+          ...FONTS.smallText,
+          color: COLORS.textBlack,  
+          marginTop: 14,
+          marginLeft: 5,
+
+      },
+      versionNumber: {
+        ...FONTS.smallText,
+        marginTop: 30,
+        marginLeft: 40,
+        color: COLORS.primary,
+      },
+      drawerElement: {
+          paddingLeft: 41,
+          color: COLORS.primary,
+      }
+  });
+
+
+  const stylesBalance = StyleSheet.create({
+    balance: {
+        width: SIZES.width * 0.75,
+        height: 101,
+        marginLeft: (SIZES.width * 0.05) / 2,
+        marginTop: -35,
+        marginBottom: 10,
+        borderRadius: 14,
+        backgroundColor: COLORS.white,
+        padding: 13,
+        justifyContent: "space-between",
+    
+        shadowColor: "#7a5dba",
+        shadowOffset: { width: 10, height: 2.5 },
+        shadowOpacity: 0.25,
+        shadowRadius: 25,
+        elevation: 100
+      },
+    
+      text: {
+        ...FONTS.h5,
+      },
+    
+      ksh: {
```

Comment:
@Chegemartinx @Allen-Muhani @AlphonceMutebi which font style s this?
It should have to be set to "bold" here. There should be one style

Reply:
Bold is specific to the use case/context.